### PR TITLE
Add PL/pg SQL (without test support)

### PR DESCRIPTION
**Pull request by GradedJestRisk**
PL/pg SQL is the procedural language for PostgreSQL DB, like PL/SQL for Oracle DB.
This proposal is bare metal, without  any test framework, but it works !

### CODE REVIEW

File: plpgsql/test/data/load.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+-- Usual item
```
Comment (emilybache) [inline plpgsql/test/data/load.sql]: These 'Usual' items unfortunately don't fit the theme of this exercise. Could you change all these items of test data to ones that match the other translations? For example in the Java version (https://github.com/emilybache/GildedRose-Refactoring-Kata/blob/master/Java/src/test/java/com/gildedrose/TexttestFixture.java) you have:

new Item("+5 Dexterity Vest", 10, 20), //
                new Item("Aged Brie", 2, 0), //
                new Item("Elixir of the Mongoose", 5, 7), //
                new Item("Sulfuras, Hand of Ragnaros", 0, 80), //
                new Item("Sulfuras, Hand of Ragnaros", -1, 80),
                new Item("Backstage passes to a TAFKAL80ETC concert", 15, 20),
                new Item("Backstage passes to a TAFKAL80ETC concert", 10, 49),
                new Item("Backstage passes to a TAFKAL80ETC concert", 5, 49),
                // this conjured item does not work properly yet
                new Item("Conjured Mana Cake", 3, 6) };

**Comment by emilybache** [comment]
Thankyou for submitting this new translation, I'm really happy to see another language here. Just bare metal, without test framework, is ok although it would be even better with that too. However, I wonder if you could just adjust the list of items as I mentioned in my other comment? I'd like that changed before I accept this pull request.

**Comment by GradedJestRisk** [comment]
Hi Emily,

Thanks for this feedback. I didn't know about standard test data, I'll fix
it today.
I planned to add test framework support (TAP), but I thought it would be
better to start small with a first PR, and do another with TAP afterwards
:°)

PL/SQL and bros have huge codebase in Europe but there's a lack of
refactoring skills on these languages.
I and other "XP" people from @OCTOTechnology are trying to create kata
targeted on this.
The real problem is missing "real" codebase to get dirty hands on.

If you know people that would accept to publish PL/SQL codebase/data for
educational purpose, let me know here 🙂
https://twitter.com/JestRisk/status/1205400139055206401

Regards,
Pierre

**Comment by GradedJestRisk** [comment]
I didn't know about these standard test data..
I have updated the actual items accordingly.

**Comment by GradedJestRisk** [comment]
I also added a SQL kata: you'll see that code weirdnesses pops up nicely this way.
I used ISO SQL everywhere possible to enable use in any database.

**Comment by emilybache** [comment]
Thankyou!

**Pull request merged**